win32ss/user/ntuser/main.c

```diff
@@ -1003,7 +1003,7 @@ DriverEntry(
     /* Create the global USER heap */
     GlobalUserHeap = UserCreateHeap(&GlobalUserHeapSection,
                                     &GlobalUserHeapBase,
-                                    1 * 1024 * 1024); /* FIXME: 1 MB for now... */
+                                    1 * 1024 * 1024); /* FIXME: 4 MB for now... */
```

Comment:
is it just me or does the new comment contradict the code?

Reply:
Right. Anyway, first commit does not belong here..